include/vara/Feature/FeatureSourceRange.h

```diff
@@ -96,14 +102,15 @@ class FeatureSourceRange {
   }
 
   inline bool operator==(const FeatureSourceRange &Other) const {
-    return Path == Other.Path and Start == Other.Start and End == Other.End;
+    return Category == Other.Category and Path == Other.Path and Start == Other.Start and End == Other.End;
   }
 
   inline bool operator!=(const FeatureSourceRange &Other) const {
     return !(*this == Other);
   }
 
 private:
+  Category Category;
   fs::path Path;
   std::optional<FeatureSourceLocation> Start;
   std::optional<FeatureSourceLocation> End;
```

Comment:
```suggestion
  fs::path Path;
  std::optional<FeatureSourceLocation> Start;
  std::optional<FeatureSourceLocation> End;
  Category Category;
```
Initialization order in the ctor needs to be same as the member order.